skgstat/Variogram.py

```diff
@@ -2111,6 +2111,103 @@ def to_gstools(self, **kwargs):
         """
         return skgstat_to_gstools(self, **kwargs)
 
+    def to_krige(self, **kwargs):
```

Comment:
Maybe a more descriptive name? `to_gs_krige`? `to_gstools_krige`? But I am fine with this also. Just to make it clear for the user, that this is an external kriging routine here.

Reply:
Agree. Makes sense. I'll go for `to_gs_krige`